CODEOWNERS

```diff
@@ -7,32 +7,32 @@
 /task/acs-image-check                       @konflux-ci/build-maintainers
 /task/acs-image-scan                        @konflux-ci/build-maintainers
 /task/apply-tags                            @konflux-ci/build-maintainers
-/task/build-image-index                     @konflux-ci/build-maintainers
-/task/build-image-manifest                  @konflux-ci/build-maintainers
-/task/buildah                               @konflux-ci/build-maintainers
-/task/buildah-min                           @konflux-ci/build-maintainers
-/task/buildah-oci-ta                        @konflux-ci/build-maintainers
-/task/buildah-remote                        @konflux-ci/build-maintainers
-/task/buildah-remote-oci-ta                 @konflux-ci/build-maintainers
-/task/buildah-rhtap                         @konflux-ci/build-maintainers
-/task/download-sbom-from-url-in-attestation @konflux-ci/build-maintainers
+/task/build-image-index                     @konflux-ci/build-maintainers @konflux-ci/the-collective
+/task/build-image-manifest                  @konflux-ci/build-maintainers @konflux-ci/the-collective
+/task/buildah                               @konflux-ci/build-maintainers @konflux-ci/the-collective
+/task/buildah-min                           @konflux-ci/build-maintainers @konflux-ci/the-collective
+/task/buildah-oci-ta                        @konflux-ci/build-maintainers @konflux-ci/the-collective
+/task/buildah-remote                        @konflux-ci/build-maintainers @konflux-ci/the-collective
+/task/buildah-remote-oci-ta                 @konflux-ci/build-maintainers @konflux-ci/the-collective
+/task/buildah-rhtap                         @konflux-ci/build-maintainers @konflux-ci/the-collective
+/task/download-sbom-from-url-in-attestation @konflux-ci/build-maintainers @konflux-ci/the-collective
```

Comment:
Is this just so that you can approve the Mobster updates or do you actually plan to co-own the tasks with us?

If it's just for Mobster, I'm not sure how I feel about it. Maybe give permissions to just one or two people who are aware they shouldn't be approving other changes? Or, if the whole team understands that, at least add a comment about it.